ha/core/event_manager/event_manager.py

```diff
@@ -384,26 +384,26 @@ def get_events(self, component : SUBSCRIPTION_LIST) -> list:
                     break
         return value
 
-    def publish(self, event: Event) -> None:
+    def publish(self, event: HealthEvent) -> None:
```

Comment:
class has changed from Event to HealthEvent 
Please confirm that this change has not broken any existing  functionality

Reply:
In cortx-utils Event class is generic , HealthEvent is specific to HA ,where it inherited by Event class but has attributes of HA health realted.